src/components/CytoscapeGraph/CytoscapeGraph.tsx

```diff
@@ -267,6 +306,21 @@ export class CytoscapeGraph extends React.Component<CytoscapeGraphProps, Cytosca
     }
   }
 
+  private handleDoubleTap = (event: CytoscapeClickEvent) => {
+    if (!event.summaryTarget.data('isOutsideNamespace')) {
+      return;
+    }
+
+    this.context.router.history.push(
```

Comment:
I think this is going to be replaced by a redux state here: https://github.com/kiali/kiali-ui/pull/435 but not really sure

@mtho11 might know more.

Reply:
Also, I think this needs to be handled outside, in the [ServiceGraphPage](https://github.com/kiali/kiali-ui/blob/master/src/pages/ServiceGraph/ServiceGraphPage.tsx#L89)
If so, this would only need to do a `this.props.onDoubleClick(event);`

Reply:
Yeah, it's not reduxified yet, so not sure how the placeholder code should be.